src/diffusers/loaders.py

```diff
@@ -991,40 +1027,29 @@ def _modify_text_encoder(self, attn_processors: Dict[str, LoRAAttnProcessor]):
         """
 
         # First, remove any monkey-patch that might have been applied before
-        self._remove_text_encoder_monkey_patch()
+        cls._remove_text_encoder_monkey_patch(text_encoder)
```

Comment:
`_remove_text_encoder_monkey_patch()` also needs to be modified no? Because we no longer have an "old_forward()" variable per your changes.

https://github.com/huggingface/diffusers/blob/ce5504934ac484fca39a1a5434ecfae09eabdf41/src/diffusers/loaders.py#L979

Reply:
It is! See here 

https://github.com/huggingface/diffusers/pull/3778/files#diff-c54108820590e5247c5a7bd1aa43b68d53a40bc630f0791cb537047752520fabR1004-R1017